pkg/utils/kubernetes/leaderelection_test.go

```diff
@@ -136,7 +136,7 @@ var _ = Describe("LeaderElection", func() {
 
 		Context("configmaps lock", func() {
 			BeforeEach(func() {
-				lock = resourcelock.ConfigMapsResourceLock
+				lock = resourcelock.ConfigMapsLeasesResourceLock
```

Comment:
Instead of using `ConfigMapsLeasesResourceLock` just don't use any constant (since there is none anymore) and write "configmaps" explicitly here.